src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -317,6 +328,9 @@ export const useTradeExecution = (
                 wallet,
               })
 
+              if (!receiverAddress) throw Error('Missing receiver address')
```

Comment:
nitpick - we are guaranteed the `receiverAddress` now with the top level `isExecutableTrade` and can omit the runtime sender/from address checks as well

Reply:
🧠  
- removed for `receiverAddress` in b1baf3d8e9
- removed `from` check in 5b02fc8756 - this one was useless either way, `adapter.getAddress()` would throw and can't return falsy